cloudant-http/build.gradle

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+dependencies {
+    compile group: 'com.google.code.gson', name: 'gson', version: '2.2.4'
+    compile group: 'commons-io', name: 'commons-io', version: '2.4'
+    //needed to compile, but optional for consumers of java-cloudant
+    compile(group: 'com.squareup.okhttp', name: 'okhttp-urlconnection', version: '2.5.0') {
+        ext.optional = true
+    }
+}
+
+javadoc {
+    include("**/http/*")
```

Comment:
Our javadoc is set to protected by default. The `Version` exclude here is because that interface is public for java-cloudant and sync-android to implement, but it isn't external API so isn't documented as such. Otherwise the `**/http/*` include brings in the non-internal classes.